src/main/java/com/porotov/articleservice/config/DataConfig.java

```diff
@@ -0,0 +1,46 @@
+package com.porotov.articleservice.config;
+
+import jakarta.annotation.Resource;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.env.Environment;
+import org.springframework.jdbc.datasource.DriverManagerDataSource;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+
+import javax.sql.DataSource;
+
+/**@author Alexandr Porotov
+ * @version  pre-alpha
+ * <p>Конфигурация базы данных</p>*/
+@Configuration
+@EnableTransactionManagement
+public class DataConfig {
```

Comment:
Do you really need this class? Spring Boot autoconfigure database from `app.properties` file via `spring-boot-starter-data-jpa`.